### PR TITLE
Port Spec Melspec

### PR DESCRIPTION
**Pull request by khumairraj**
Following are the changes - 
1. Port the Spec and Mel spec from kapre 0.1.4
2. Add weights for Mel from openl3.
3. Fix the resampling error due to use of julian. This is an option which we can pass now into the model.
4. Since resampy works on numpy weights the beginning part required to have 64 bit tensor and then scale down to 32 bit for further processing as in openl3.

### CODE REVIEW

File: tests/test_embedding_timestamps.py [hunk 1]
```diff
@@ -41,11 +41,11 @@ def test_timestamps_no_channels(self):
 
         for params in paramlist:
             emb0, ts0 = torchopenl3.get_audio_embedding(
-                sounds, 48000, batch_size=32, **params
+                sounds, 48000, batch_size=32, sampler="resampy", **params
```
Comment (turian) [inline tests/test_embedding_timestamps.py:44]: Why don't we just make "resampy" the default in get_audio_embedding, and make a note about why it's the default?

File: torchopenl3/core.py [hunk 2]
```diff
@@ -99,6 +110,7 @@ def get_audio_embedding(
     batch_size=32,
     verbose=True,
     weight_path="",
+    sampler="julian",
```
Comment (turian) [inline torchopenl3/core.py:113]: Let's remove this then? Since it will give the wrong output

File: tests/test_embedding_timestamps.py (hunk 1 above)
Comment (khumairraj) [reply tests/test_embedding_timestamps.py:44]: yes we can do that. its just that julius is based on pytorch, so kept this as an option.

Comment (turian) [reply tests/test_embedding_timestamps.py:44]: No if Julius gives bad results then we don't want to include it in this release

Comment (khumairraj) [reply tests/test_embedding_timestamps.py:44]: sure. then, will change that the preprocessing part to numpy. I think that will be fine.

**Pull request merged**